website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

```diff
@@ -206,15 +199,17 @@ the following steps.
      the hostname is `https://app.terraform.io`. Be sure to include "https://"
      before the hostname.
    - Set API Team Token to the Terraform Cloud team API token.
+   - (Optional) To use the [MID Server](https://docs.servicenow.com/bundle/vancouver-servicenow-platform/page/product/mid-server/concept/mid-server-landing.html), 
```

Comment:
@sheneska please update the URL to not contain "vancouver" in it as it will break sometime in the future when Vancouver is deprecated. Whenever we share SNOW documentation links, let's use the "share" button > "Copy latest version URL" to copy a link that is not tied to any specific version.